helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/MetadataStoreDirectoryAccessor.java

```diff
@@ -0,0 +1,163 @@
+package org.apache.helix.rest.server.resources.zookeeper;
```

Comment:
We should put this in a different package. 

How about `...server.resources.metadatastore;` ?